lib/src/builders/message/message.dart

```diff
@@ -122,11 +132,13 @@ class MessageBuilder extends CreateBuilder<Message> {
           'message_id': replyId.toString(),
           if (requireReplyToExist != null) 'fail_if_not_exists': requireReplyToExist,
         },
+      if (components != null) 'components': components!.map((e) => e.build()).toList(),
       if (stickerIds != null) 'sticker_ids': stickerIds!.map((e) => e.toString()).toList(),
       if (attachments != null) 'attachments': attachments!.map((e) => e.build()).toList(),
-      if (suppressEmbeds != null || suppressNotifications != null)
-        'flags':
-            (suppressEmbeds == true ? MessageFlags.suppressEmbeds.value : 0) | (suppressNotifications == true ? MessageFlags.suppressNotifications.value : 0),
+      if (suppressEmbeds != null || suppressNotifications != null || isEphemeral != null)
+        'flags': (suppressEmbeds == true ? MessageFlags.suppressEmbeds.value : 0) |
+            (suppressNotifications == true ? MessageFlags.suppressNotifications.value : 0) |
+            (isEphemeral == true ? MessageFlags.ephemeral.value : 0),
```

Comment:
`== true` could be removed. just `suppressEmbeds` gonna be sufficient

Reply:
The fields are nullable, so it'd have to be `suppressEmbeds != null ? suppressEmbeds! : false` which is longer.